client/packages/system/src/Manage/Preferences/Components/EditWarningWhenMissingRecentStocktakeData.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {
+  Box,
+  Typography,
+  NumericTextInput,
+  Switch,
+  WarnWhenMissingRecentStocktakeDataNode,
+  InputWithLabelRow,
+  useTranslation,
+} from '@openmsupply-client/common';
+
+// Backend returns snake_case, but TypeScript types expect camelCase
+type WarnWhenMissingRecentStocktakeDataSnakeCase = {
```

Comment:
This is the messiest thing... Not sure why it's `snake_case` in one context and `camelCase` in the other...